omod/src/main/webapp/uiError.jsp

```diff
@@ -1,12 +1,7 @@
-<%@ include file="/WEB-INF/template/include.jsp"%>
-<%@ include file="/WEB-INF/template/header.jsp"%>
-
 <h1>UI Framework Error</h1>
```

Comment:
Very easy to test. You can just run the webapp in debug mode and put a break point in this class. Then stop or start or install or uninstall any module.

You could as well just enclose it in a try catch handler and printStackTrace as done here: https://github.com/openmrs/openmrs-module-legacyui/blob/master/omod/src/main/java/org/openmrs/web/WebComponentRegistrar.java#L57-L64